jobs/JEVS_WAFS_ATMOS_PLOTS

```diff
@@ -68,7 +68,8 @@ export COMINstat=$COMIN/stats/$COMPONENT  # will be appended with wafs.$VDATE or
 
 # COMOUT folder
 # evs_ver defined in versions/run.ver
-export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})/$STEP/$COMPONENT/$RUN.$VDATE}
+export COMROOT=${COMROOT:-$(compath.py -o ${NET}/${evs_ver})}
+export COMOUT=$COMROOT/$STEP/$COMPONENT/$RUN.$VDATE
```

Comment:
@YaliMao-NOAA COMOUT need to be defined using "compath.py -o" in operational code. Please bring back the COMOUT line that you deleted and remove the COMROOT and new COMOUT lines. We want:

export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})/$STEP/$COMPONENT/$RUN.$VDATE}

This line should work with the COMROOT you included in your WAFS plot driver script.

This should work in your current set up because